target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -150,18 +132,10 @@
 	};
 };
 
-&i2c {
-	status = "okay";
-};
-
 &uart1 {
 	status = "okay";
 };
 
-&pwm {
-	status = "okay";
-};
-
```

Comment:
Why is this one removed?

Reply:
it was mentioned somewhere before, that if i cannot test the functionality it should not be in the feature node. As the pin is only available with someones own mother board, i currently have no possibility to test it, as it is not available on the evalboard. Normally i would leave it in, because i'd assume it should work.

Reply:
Since this is about changing an existing state, I'd only _change_ it when verified.